tests/examples/test_dataset_loader.py

```diff
@@ -69,10 +69,8 @@ def add_to_dict(func: str, dataset_function: Callable[[], Any]):
 
     # Collect all `download_<name> or `load_<name>` functions
     def _is_dataset_function(name, item):
-        return (
-            isinstance(item, FunctionType)
-            and name.startswith('download_')
-            or name.startswith('load_')
+        return (isinstance(item, FunctionType) and name.startswith('download_')) or name.startswith(
+            'load_'
```

Comment:
I ran this locally and double checked that test_examples and test_downloads are working as expected w.r.t. this function
```suggestion
        return isinstance(item, FunctionType) and name.startswith(('download_', 'load_'))
```